scripted-sbt-redux/src/main/scala/sbt/scriptedtest/ScriptedTests.scala

```diff
@@ -482,102 +403,62 @@ object ScriptedTests extends ScriptedRunner {
 class ScriptedRunner {
   // This is called by project/Scripted.scala
   // Using java.util.List[File] to encode File => Unit
-  def run(
-      resourceBaseDirectory: File,
-      bufferLog: Boolean,
-      tests: Array[String],
-      bootProperties: File,
-      launchOpts: Array[String],
-      prescripted: java.util.List[File],
-  ): Unit = {
-    val logger = new TestConsoleLogger()
-    val addTestFile = (f: File) => { prescripted.add(f); () }
-    run(resourceBaseDirectory, bufferLog, tests, logger, bootProperties, launchOpts, addTestFile)
-    //new FullLogger(Logger.xlog2Log(log)))
-  }
-
-  // This is called by sbt-scripted 0.13.x and 1.x (see https://github.com/sbt/sbt/issues/3245)
-  def run(
-      resourceBaseDirectory: File,
-      bufferLog: Boolean,
-      tests: Array[String],
-      bootProperties: File,
-      launchOpts: Array[String],
-  ): Unit = {
-    val logger = TestConsoleLogger()
-    val prescripted = ScriptedTests.emptyCallback
-    run(resourceBaseDirectory, bufferLog, tests, logger, bootProperties, launchOpts, prescripted)
-  }
```

Comment:
Because of `sbt ^scripted`, we need this entry point - https://github.com/sbt/sbt/issues/3245